player/command.c

```diff
@@ -560,7 +561,10 @@ static int mp_property_stream_open_filename(void *ctx, struct m_property *prop,
     }
     case M_PROPERTY_GET_TYPE:
     case M_PROPERTY_GET:
-        return m_property_strdup_ro(action, arg, mpctx->stream_open_filename);
+        path = mp_normalize_path(NULL, mpctx->stream_open_filename);
```

Comment:
Dumb nit but why not just `char *path` here?

Reply:
Thanks.